### DISCUSSION TITLE
How to register tokens globally?

### DISCUSSION THREAD

**Discussion by azamat7g** [beholdr/maska#212] [Q&A] [answered]
I need some new tokens globally. In old version (v1) was 8 preregistered tokens. In v2 there are only 3. I use nuxt3. I'm registered vMask directive. But i can't find how to register new tokens globally. Please help

**Comment by beholdr** [comment, marked as answer]
There is no global tokens registration in v2. You can create custom tokens with `data-maska-tokens` attribute or pass them via options. In second case you can create some global variable and use it for every case.
